In [ ]:
# Das Paket tidyverse wird geladen
library(tidyverse);

load(file = "Unterschiedshypothesen1.RData")

## Aufgabe 1.
Damit wir von einer Vergleichbarkeit der beiden Interventionsgruppen ausgehen können, möchten wir überprüfen, ob sich die verschachtelte und die geblockte Bedingung (gruppe) in ihrem Vorwissen (vorwissen) unterscheiden. Überprüft dies.

**Als Statistisches Verfahren wird der t-Test für unabhängige Stichproben bzw. der Welch Test gewählt.**


In [10]:
psych::describeBy(x = Unterschiedshypothesen1$vorwissen, 
                  group = Unterschiedshypothesen1$gruppe)


 Descriptive statistics by group 
group: 0
   vars  n  mean   sd median trimmed  mad min max range  skew kurtosis   se
X1    1 98 13.82 5.78   14.5   14.04 6.67   0  25    25 -0.28    -0.63 0.58
------------------------------------------------------------ 
group: 1
   vars  n  mean   sd median trimmed  mad min max range skew kurtosis  se
X1    1 95 14.12 5.87     14   14.06 7.41   0  26    26 0.04    -0.79 0.6

Empirische Kennwerte: 
verschachtelt: M = 13.82, SD = 5.78
geblockt: M = 14.12, SD = 5.87

In [8]:
#
t.test(vorwissen ~ gruppe, data=Unterschiedshypothesen1)


	Welch Two Sample t-test

data:  vorwissen by gruppe
t = -0.35709, df = 190.57, p-value = 0.7214
alternative hypothesis: true difference in means is not equal to 0
95 percent confidence interval:
 -1.953630  1.354704
sample estimates:
mean in group 0 mean in group 1 
       13.81633        14.11579 


**Ergebnis**

t(191) = -0.36, p = .72, d = -0.05

**Interpretation:** Es gibt keinen signifikanten Unterschied im Vorwissen zwischen der
verschachtelten und der geblockten Bedingung – damit sind die beiden Interventionsgruppen
bezüglich ihres Vorwissens vergleichbar. 